src/sentry/incidents/models/alert_rule.py

```diff
@@ -475,7 +475,9 @@ class AlertRuleTriggerAction(AbstractNotificationAction):
     alert_rule_trigger = FlexibleForeignKey("sentry.AlertRuleTrigger")
 
     date_added = models.DateTimeField(default=timezone.now)
-    sentry_app_config = JSONField(null=True)
+    sentry_app_config = JSONField(
+        null=True
+    )  # list of dicts if this is a sentry app, otherwise can be singular dict
```

Comment:
for documentation purposes -- i don't think this should be a list of dicts, if you can just collapse them into one... example usage looks like this
https://github.com/getsentry/sentry/blob/2e8743f18ad02ab4a328c7e4ca1d09498da42896/tests/sentry/incidents/action_handlers/test_sentry_app.py#L118-L124